src/rpc/services.h

```diff
@@ -25,10 +25,6 @@ class NodeService {
     virtual void HandleRequestHeartbeat(RpcConn conn, int32_t term_id,
                                         std::string node_id_binary) = 0;
 
-    virtual void HandleRequestPullLogs(RpcConn conn, bool result,
-                                       std::string node_id_binary,
-                                       int64_t next_log_index) = 0;
-
     virtual void HandleRequestPushLogs(RpcConn conn, int64_t committed_log_index,
```

Comment:
PushLogs has no return value. How does the leader know the result or the committed index of this RPC request?

Before this PR, PullLogs can tell the leader the information.

Reply:
> PushLogs has no return value. How does the leader know the result or the committed index of this RPC request?
> 
> Before this PR, PullLogs can tell the leader the information.

In HandleRequestPushLogs, followers should response with a success flag which will let leader knows if this RPC request succeeds.

Reply:
I don't think it does work. Actually, handling push logs requests in follower is async operation. So the return value `success` doesn't indicates if the follower has consumed the requests.